### PR TITLE
specify tested keras version in readme

### PR DESCRIPTION
**Pull request by cocuh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ Model compression CLI tool for [keras](https://github.com/fchollet/keras).
 
 ## Requirements
 - Python 3.5, 3.6
-- TensorFlow backend
+- keras
```
Comment (henry0312) [inline README.md]: Keras

File: README.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ Model compression CLI tool for [keras](https://github.com/fchollet/keras).
...
+    - We tested keras 2.0.3 (TensorFlow backend)
```
Comment (henry0312) [inline README.md]: We tested [on or with] Keras 2.0.3 (TensorFlow backend)

**Review by henry0312 — CHANGES_REQUESTED** [review]
very tiny fix

**Comment by cocuh** [comment]
fixed @henry0312

**Review by henry0312 — APPROVED** [review]

**Pull request merged**